docs/milestones.md

```diff
@@ -0,0 +1,61 @@
+# Gestión de Medicamentos - Milestones
+
+## **[M0] Milestone 0: Modelado y planificación del dominio del problema**
+Entregar un modelo inicial del dominio que defina claramente las entidades clave y sus relaciones, basado en las historias de usuario. Este modelo incluirá conceptos como medicamentos, tickets electrónicos (PDF, XML) e inventario.
+
+**Producto**:
+- Código fuente que represente el modelo del dominio del problema, basado en las historias de usuario [HU1] y [HU2].
+- Diagramas UML que describan las relaciones entre las entidades clave y los procesos involucrados en la gestión de medicamentos y tickets electrónicos.
+  
+**Factibilidad**:  
+Se considerará factible cuando el código fuente entregado represente correctamente el dominio del problema en función de la información recopilada en las historias de usuario antes mencionadas.
```

Comment:
Non si parla di fattibilità (cualunque sia ciò) ma di "vitalità" o "funzionalitá", cioè, che funciona: prodotto minimo funzionante. Quindi, devi dire quà "Come si sa che funziona?" "Factible" in spagnuolo e "che si può fare". Se è fatto, ecco, è fattibile